includes/admin/post-types.php

```diff
@@ -140,17 +140,15 @@ public function row_actions($actions, $post)
 	{
 		// Add a clear feed cache action link.
 		if ($post->post_type == 'calendar' && current_user_can('edit_posts')) {
-			$nonce_feed_actions = wp_create_nonce('nonce_feed_actions');
-
 			$actions['duplicate_feed'] =
 				'<a href="' .
-				esc_url(add_query_arg(['duplicate_feed' => $post->ID, 'nonce_feed_actions' => $nonce_feed_actions])) .
```

Comment:
@sujitxtendify Please take a pull and push changes. you replace changes done in a recent release.